dspace/config/modules/solr-statistics.cfg

```diff
@@ -28,10 +28,10 @@ solr-statistics.configset = statistics
 solr-statistics.autoCommit = true
 
 # URLs to download IP addresses of search engine spiders from
-solr-statistics.spiderips.urls = http://iplists.com/google.txt, \
-                 http://iplists.com/inktomi.txt, \
-                 http://iplists.com/lycos.txt, \
-                 http://iplists.com/infoseek.txt, \
-                 http://iplists.com/altavista.txt, \
-                 http://iplists.com/excite.txt, \
-                 http://iplists.com/misc.txt
+solr-statistics.spiderips.urls = https://iplists.com/google.txt, \
```

Comment:
Silly question, @kshepherd ...  why is this the only URL that starts with `https://iplists.com` instead of `https://www.iplists.com`?  Should it be updated too?

Beyond that, this is all small/obvious fixes.  Looks good